src/Eccube/Service/Composer/ComposerProcessService.php

```diff
@@ -75,8 +75,8 @@ public function execRemove($packageName)
         $this->init();
         // Build command
         $command = $this->getPHP().' '.$this->composerFile.' remove '.$packageName;
-        $command .= ' --no-progress --no-scripts --ignore-platform-reqs --profile --no-ansi --no-interaction -d ';
-        $command .= $this->workingDir.' 2>&1';
+        $command .= ' --no-progress --no-scripts --ignore-platform-reqs --profile --no-ansi --no-interaction --no-update-with-dependencies -d ';
+        $command .= $this->appConfig['root_dir'].' 2>&1';
```

Comment:
why change here ?

Reply:
It may be a mistake to resolve the conflict